odbc/src/api.rs

```diff
@@ -1115,12 +1144,55 @@ pub extern "C" fn SQLSetEnvAttr(
 
 #[no_mangle]
 pub extern "C" fn SQLSetEnvAttrW(
-    _environment_handle: HEnv,
-    _attribute: EnvironmentAttribute,
-    _value: Pointer,
+    environment_handle: HEnv,
+    attribute: EnvironmentAttribute,
+    value: Pointer,
     _string_length: Integer,
 ) -> SqlReturn {
-    unimplemented!()
+    let env_handle = MongoHandleRef::from(environment_handle);
+    env_handle.clear_diagnostics();
+    match env_handle.as_env() {
+        None => SqlReturn::INVALID_HANDLE,
+        Some(env) => match attribute {
+            EnvironmentAttribute::OdbcVersion => match value as Integer {
+                380 => {
+                    let mut env_contents = (*env).write().unwrap();
+                    env_contents.attributes.odbc_ver = value as _;
+                    SqlReturn::SUCCESS
+                }
+                _ => {
+                    env_handle.add_diag_info(ODBCError::InvalidAttrValue("SQL_ATTR_ODBC_VERSION"));
+                    SqlReturn::ERROR
+                }
+            },
+            EnvironmentAttribute::ConnectionPooling => match value as Integer {
+                0 => SqlReturn::SUCCESS,
+                _ => {
+                    env_handle
+                        .add_diag_info(ODBCError::Unimplemented("SQL_ATTR_CONNECTION_POOLING"));
+                    SqlReturn::ERROR
+                }
+            },
+            EnvironmentAttribute::CpMatch => match value as Integer {
```

Comment:
Since we don't support pooling, I think that none of the option are valid, no?
But again, this looks like a DM related setting, so maybe it's just deadcode.